boards/arm/ip_k66f/Kconfig.defconfig

```diff
@@ -20,6 +20,19 @@ config MCG_VDIV0
 config MCG_FCRDIV
 	default 1
 
+if NETWORKING
+
+config NET_L2_ETHERNET
+	default y if !MODEM
```

Comment:
Do you have MODEM on this board?

Reply:
No, there is no modem. I think that it was a leftover from frdm_k64f board. I will remove the !MODEM